### ISSUE TITLE
Prefixed by default?

### ISSUE THREAD

**Issue by jaredcwhite** [tokencss/tokencss#16]
First off, very intriguing project! I'll enjoy seeing how this progresses.

There was a Twitter thread about this but I didn't see it captured in an issue so wanted to add it…essentially that having some kind of prefix on the tokens by default would make it clear this is a non-standard token syntax within the CSS code. The idea that `background: rebeccapurple` is vanilla CSS but `background: primary` gets processed into something else feels strange to me, whereas something like `background: $primary` feels obvious that I'm looking at an abstracted token.

If not by default, then at least an option would be great (and apologies if this is already a supported feature).

**Comment by natemoo-re** [comment]
Yep, thanks for opening an issue so this doesn't get lost. I am planning on switching to `$` prefixes by default in the next minor version for multiple reasons:

- Easier to distinguish what is a token value at a glance
- Almost guaranteed back/forward compat (CSS spec won't go near `$` due to Sass)
- Better IDE support (typing `$` will only suggest token values)

**Comment by jaredcwhite** [comment]
@natemoo-re Awesome to hear that! 👍

**Comment by romainmenke** [comment]
> Almost guaranteed back/forward compat (CSS spec won't go near $ due to Sass)

Is this not a good reason not to use it 🤔 
Picking `$` makes it completely impossible to have a file with both `scss` and tokencss.

--------------

We focussed on a maximally compatible grammar in source and using IDE features to have fast typing short cuts.

typing `smal..` can expand to `design-token('space.small')`

https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-design-tokens#readme
